### PR TITLE
update workload for acm 2.10

### PR DESCRIPTION
**Pull request by haoqing0110**

### CODE REVIEW

File: deploy/generate.sh [hunk 1]
```diff
@@ -63,7 +64,8 @@ echo "managed cluster creationTimestamp: $mc_timestamp"
...
-create_yaml_by_offset "$mc_timestamp" "10.5" "12" "job_enable_app"
-create_yaml_by_offset "$mc_timestamp" "12" "13.5" "job_enable_policy_proxy"
-create_yaml_by_offset "$mc_timestamp" "13.5" "15" "job_enable_policy_search"
-create_yaml_by_offset "$mc_timestamp" "15" "16.5" "job_enable_all"
\ No newline at end of file
+# uncomments below code when testing benchmark for addon combinations
+#create_yaml_by_offset "$mc_timestamp" "10.5" "12" "job_enable_app"
+#create_yaml_by_offset "$mc_timestamp" "12" "13.5" "job_enable_policy_proxy"
+#create_yaml_by_offset "$mc_timestamp" "13.5" "15" "job_enable_policy_search"
+#create_yaml_by_offset "$mc_timestamp" "15" "16.5" "job_enable_all"
```
Comment (zhujian7) [inline deploy/generate.sh:71]: EOF

File: deploy/kustomization.yaml [hunk 2]
```diff
@@ -1,12 +1,2 @@
...
-- ./output/job_app_delete.yaml
-- ./output/job_policy_create.yaml
-- ./output/job_policy_delete.yaml
-- ./output/job_obs_create.yaml
-- ./output/job_obs_delete.yaml
-- ./output/job_enable_all_create.yaml
-- ./output/job_enable_app_create.yaml
-- ./output/job_enable_policy_search_create.yaml
-- ./output/job_enable_policy_proxy_create.yaml
+- cluserrolebinding.yaml
```
Comment (zhujian7) [inline deploy/kustomization.yaml:2]: ditto

Comment (haoqing0110) [reply deploy/kustomization.yaml]: fixed

File: deploy/generate.sh (hunk 1 above)
Comment (haoqing0110) [reply deploy/generate.sh]: fixed

**Pull request merged**